packages/agoric-cli/src/deploy.js

```diff
@@ -231,11 +232,12 @@ export { bootPlugin } from ${JSON.stringify(absPath)};
 
           // Use Node.js ESM support if package.json of template says "type":
           // "module".
-          const read = async url => fs.readFile(new URL(url).pathname);
+          const read = async location =>
+            fs.readFile(url.fileURLToPath(location));
           const { packageDescriptorText } =
             await readContainingPackageDescriptor(
               read,
-              `file://${moduleFile}`,
+              url.pathToFileURL(moduleFile),
```

Comment:
something tells me "portable" also means powerful / ambient

Reply:
It certainly at least means “sensitive to platform”. The big difference is the treatment of drive letters as absolute paths instead of concatenating them like `c:/users/kris/endo/d:/users/kris/` etc.